src/test/java/com/VetApp/PurrgentCare/services/AccountServiceImplementationTest.java

```diff
@@ -208,4 +231,71 @@ public void toggleAccount_whenAccountNotExists_throwEntityNotFoundException() {
         then(exception.getMessage()).contains(String.valueOf(accountId));
 
     }
+
+    // Test associatePeople when account exists
+        //This method adds a list of personIds to an account
+        //
+    @Test
+    public void associatePeople_whenAccountExists_returnAssociatedPeople() {
+
+        // Variables used everywhere
+        final var accountId = new Random().nextInt(1000);
+        final var personId = new Random().nextInt(1000);
+        final var active = Boolean.TRUE;
+        final var dateCreated = new Date();
```

Comment:
Were you following a specific workflow for this test or did you just clean it up as you went? Like how did you know to break it down into variables used everywhere and so on? I was able to conceptualize how we had our VARs before.

Reply:
Not a specific workflow. Just noticed we were going to be using them several times.
The peopleIdList should be moved up to here as well to be honest.
The purpose of these variables is remove duplicated code.